Task 1/index.js

```diff
@@ -0,0 +1,38 @@
+/* 
+Дан масив об'єктів. Вивести масив телефонних номерів користувачів, у яких баланс більше 2000 доларів. 
+І знайти суму всіх балансів користувачів
+*/
+
+import users from './data/data.json' assert { type: 'json' };
+import { convertDollarsToNumber } from './helpers/helper.js';
+
+const filterUserPhonesByBalance = (value, index) => {
+	const generateFilterArr = [];
+
+	if (Number.isInteger(index) && Array.isArray(value)) {
+		for (let i = 0; i < value.length; i++) {
+			const element = value[i];
+			if (convertDollarsToNumber(element.balance) >= index) {
+				generateFilterArr.push(element.phone);
+			}
+		}
+	} else {
+		throw 'TypeError: Arguments wrong type.';
```

Comment:
```suggestion
		throw new TypeError('filterUserPhonesByBalance() expects Array as first argument, and Number as second argument.');
```
and good practice is placing this condition on the start
```javascript
function add(a, b) {
  if (typeof a !== 'number' || typeof b !== 'number')  {
    throw new TypeError('add() expects two numbers as arguments.');
  }
  return a + b;
}
```